SerialPrograms/Source/PokemonHome/Programs/PokemonHome_BoxSorting.cpp

```diff
@@ -99,14 +99,14 @@ BoxSorting::BoxSorting()
         1, 1, MAX_BOXES
         )
     , VIDEO_DELAY(
-          "<b>Delay of your capture card (you probably have to increase this):</b>",
+          "<b>Delay of your capture card:</b>",
           LockMode::LOCK_WHILE_RUNNING,
-          20
+          50
           )
     , GAME_DELAY(
-          "<b>Delay of your Pokemon Home app (default value should be fine):</b>",
+          "<b>Delay of your Pokemon Home app:</b>",
           LockMode::LOCK_WHILE_RUNNING,
-          15
+          30

```

Comment:
I went the safe way because many people reoprted issues so trading longer running time for easier to reach reliability is better